superset/assets/src/explore/controls.jsx

```diff
@@ -1575,11 +1575,20 @@ export const controls = {
   },
 
   time_compare: {
-    type: 'TextControl',
+    type: 'SelectControl',
+    multi: true,
```

Comment:
I think that there's an open PR that makes this backward compatible. Here: https://github.com/apache/incubator-superset/pull/5057 , otherwise I think it would fail for previously saved charts

Reply:
Yeah, I remembered that as soon as I created the PR. Cool that there's a PR addressing it.

Reply:
Merged it, so we're good